packages/backend/test/api/controllers/events/EventsController.test.ts

```diff
@@ -0,0 +1,136 @@
+import { mock } from '@l2beat/common'
+import { Layer2Event } from '@l2beat/config'
+import { EthereumAddress, ProjectId, UnixTime } from '@l2beat/types'
+import { expect, mockFn } from 'earljs'
+
+import {
+  EventController,
+  getEndpointConfig,
+} from '../../../../src/api/controllers/events/EventsController'
+import {
+  getHourlyMinTimestamp,
+  getSixHourlyMinTimestamp,
+} from '../../../../src/api/controllers/report/charts'
+import { EventUpdater } from '../../../../src/core/events/EventUpdater'
+import { ProjectInfo } from '../../../../src/model'
+import {
+  EventGranularity,
+  EventRecord,
+  EventRepository,
+} from '../../../../src/peripherals/database/EventRepository'
+const PROJECT_A = ProjectId('project-a')
+const PROJECT_B = ProjectId('project-b')
+
+const EVENT_A = 'EventA'
+const MOCK_TYPES: ['timestamp', ...string[]] = ['timestamp', EVENT_A]
+
+const START = UnixTime.fromDate(new Date('2022-05-17'))
+
+const mockProject = (name: ProjectId): ProjectInfo => {
+  return {
+    name: name.toString(),
+    projectId: name,
+    escrows: [],
+    events: [mockEvent(EVENT_A)],
+  }
+}
+const mockEvent = (name: string): Layer2Event => {
+  return {
+    name: name,
+    abi: `event ${name}()`,
+    emitter: EthereumAddress.random(),
+    type: 'state',
+    sinceTimestamp: START.add(-365, 'days'),
+  }
+}
+
+const mockEventRecord = (
+  offset: number,
+  name: string,
+  granularity: EventGranularity,
+): EventRecord => {
+  return {
+    timestamp: START.add(offset, 'days'),
+    name,
+    projectId: ProjectId('random'),
+    count: 1,
```

Comment:
Setting the count to 1 everywhere makes it hard to see if the values are correct